superset/reports/schemas.py

```diff
@@ -84,7 +84,7 @@
     "Superset nags you again. (in seconds)"
 )
 working_timeout_description = (
-    "If an alert is staled at a working state, how long until it's state is reseted to"
+    "If an alert is staled at a working state, how long until its state is reset to"
```

Comment:
As noted in https://github.com/apache/superset/pull/19648#discussion_r847405605, I'm not generally doing grammar fixes, this was just in the neighborhood for the line.